pymc/logprob/transforms.py

```diff
@@ -232,20 +232,20 @@ def measurable_transform_logcdf(op: MeasurableTransform, value, *inputs, **kwarg
     """Compute the log-CDF graph for a `MeasurabeTransform`."""
     other_inputs = list(inputs)
     measurable_input = other_inputs.pop(op.measurable_input_idx)
-
-    # Do not apply rewrite to discrete variables
-    if measurable_input.type.dtype.startswith("int"):
-        raise NotImplementedError("logcdf of transformed discrete variables not implemented")
-
     backward_value = op.transform_elemwise.backward(value, *other_inputs)
 
     # Fail if transformation is not injective
     # A TensorVariable is returned in 1-to-1 inversions, and a tuple in 1-to-many
     if isinstance(backward_value, tuple):
         raise NotImplementedError
 
+    is_discrete = measurable_input.type.dtype.startswith("int")
```

Comment:
Shall we use `discrete_types` here?

https://github.com/pymc-devs/pymc/blob/43dc4be82a487cf2a90abd5e8165c1d3efb7ade1/pymc/vartypes.py#L32

Reply:
No strong preference